.sync.yml

```diff
@@ -1,5 +1,9 @@
 ---
 .travis.yml:
+  before_install:
+    - yes | gem update --system
+    - bundle --version
+    - git clone https://github.com/puppetlabs/puppetlabs-ruby_task_helper ../ruby_task_helper
```

Comment:
Since the ruby_task_helper is a puppet module, isn't it easier to add it to `.fixtures.yml`?
This is what I did in https://github.com/theforeman/puppet-puppet/pull/709/files#diff-94a00eef4dffb5a94b0c6739338e6757R15

Reply:
what about adding it to the metadata.json is a dependency?

Reply:
Both isn't helping for the problem I'm having: The path to the `task_helper.rb` file is different in tests because it get relative included:

```ruby
require_relative '../../ruby_task_helper/files/task_helper.rb'
```

This needs to be done because `ruby_task_helper` is currently a Puppet module and not a standalone gem. This, by the way, is the the [suggested](https://github.com/puppetlabs/puppetlabs-ruby_task_helper#usage) way of doing this.

Reply:
@alexjfisher suggested the following way on Slack:

```ruby
begin
  require_relative '../../ruby_task_helper/files/task_helper.rb'
rescue LoadError
  require_relative '../spec/fixtures/modules/ruby_task_helper/files/task_helper.rb'
end
```

This would add code only required in tests to the Bolt task. I usually try to avoid such situation on all costs. What do you think?

Reply:
As discussed with @alexjfisher on Slack: I think its better to use the .travis.yml variant because it's the one used by Puppetlabs modules. This ensure we use the same "style" as they.

Reply:
I'm fine with either approach.